src/SFA.DAS.ASK.Web/Views/RequestSupport/CheckYourDetails.cshtml

```diff
@@ -95,7 +95,7 @@ else
                     <dd class="govuk-summary-list__actions">
                         @if (Model.SupportRequestType == SupportRequestType.Manual || Model.NumberOfOrgs == DfeOrganisationCheckResult.Multiple)
                         {
-                            <a asp-action="@changeAddressAction" asp-controller="@changeAddressController" asp-route-requestId="@Model.RequestId" asp-route-edit="true" class="govuk-link">Change</a>
+                            <a asp-action="@changeAddressAction" asp-controller="@changeAddressController" asp-route-requestId="@Model.RequestId" asp-route-search="@Model.SearchTerm" asp-route-edit="true" class="govuk-link">Change</a>
```

Comment:
This change link will have to be different for the Dfe Multiple one as you don't want to pass the searchTerm to wherever the multiple orgs links goes to.